tests/test_union_inherited_body.py

```diff
@@ -4,8 +4,6 @@
 from fastapi.testclient import TestClient
 from pydantic import BaseModel
 
-from .utils import needs_py37
-
 # In Python 3.6:
```

Comment:
It seems like this comment about differences between Python 3.6 and 3.7 can be removed, too.